articles/fuzzy-logic-operations/index.md

```diff
@@ -0,0 +1,138 @@
+Fuzzy logic is an approach to solving problems that take into consideration the degrees of truth. Fuzzy Systems as a subject developed to model the uncertainty and vagueness present in the human thought process. 
+
+In this article, we will understand the following concepts.
+
+### Prerequisites
+
+This article on fuzzy logic is an excellent introduction to the basics of crisp and fuzzy sets. Moreover, it introduces the representation of fuzzy sets and their applications. 
+
+### Table of contents
+
+- Degree of membership function
+- Types of membership functions
+- Operations in fuzzy systems
+- Types of operations
+
+### Degree of membership function
+
+A fuzzy set is denoted as a tuple that includes the element and a value indicating its degree of membership to the universal set. Mathematically, it is represented as:
+
+$$ F= (S,\;\mu(S))\; ;\;S\, \epsilon\, x\; \&\; \mu(S)\:=\:Degree\, of\, S $$
+ 
+The above equation can be viewed as a collection of ordered pairs. The inclusion of the element $S \epsilon\; x$ into the fuzzy set $F$ is fuzzy.
+
+#### Example
+
+Consider the example of temperature control. We will explore the various membership functions using this example. The task to control the room temperature can be modeled as a fuzzy problem. Let the ranges of temperature be given as follows:
+
+$$
```

Comment:
For some reason, VSCode Markdown renders this line without line breaks

![image](https://user-images.githubusercontent.com/34317373/105633442-9abe8700-5e7e-11eb-9340-6be3d16bfd8c.png)

Reply:
This is how I get it. @hectorkambow Can you check once on your end? The same Latex code is rendering differently on different machines. 
![image](https://user-images.githubusercontent.com/26842636/105848209-ac319b80-6004-11eb-865b-e922ab9a952e.png)